wiki/Multi/en.md

```diff
@@ -0,0 +1,487 @@
+
+#### Result, Retry, and Replay
+
+Despite no online score leaderboard below the results screen, ranked score will still be counted as legitimate ranked entry.
+Result will not be shown at the Local leaderboard.
+The only way to see it is by Online Ranking leaderboard (assuming this score is higher).
+
+**No direct retry allowed** but the replay can be exported by pressing `F2` key in keyboard (will not contain any Multi-play elements).
+Do note that this does not apply to Tag Coop and Tag Team Vs (both are unrankable), where score will not be recorded and replay cannot be saved.
+
+#### Multi's leaderboard
+
+The leaderboard now shows the players in the match with live placements changes based on Win Condition as shown below:-
+
+**Head to Head / Team Vs:-**
+
+- If Score/Score v2: Score, hit score and combo multiplier are shown in real-time.
+- If Accuracy: Accuracy, hit score and combo multiplier are shown in real-time.
+- If Combo: Combo count are shown in real-time. Maximum combo count not shown.
+
+**Tag Coop / Tag Team Vs:-**
+
+- If Score/Score v2: Team's score and hit score are shown in real-time.
+- If Accuracy: Team's accuracy percentage and hit score are shown in real-time.
+- If Combo: Team's current combo count, score and hit score are shown in real-time.
+- If _Failed_: Match ends and (Tag Team Vs only) automated win for the surviving team.
+- If _Quit_: [Auto][Auto wikilink] will take over the _Quit_ player.
+
+#### Player box colour
+
+<!-- A player box reference here? Old images in ./img/Playerbox -->
+<!-- Special player box note: Skipped (has its own special callout), Failed (red text), and Quit (red text with [Quit] appended) -->
+
+Status/Colour | Description
+:---:|:---
+**Normal/Blue**       | **Player has _more_ than half of health**, usually at full. The concentration of blue colour will grow brighter if the player's health bar is increasing to full and bleed to red by losing the health bar more than half of the health bar length.
+**Danger/Purple-Red** | **Player has _less_ than half of health**. The concentration of red colour will grow brighter if the player's health bar is decreasing to empty and diminished to blue by filling the health bar more than half the health bar length.
+**Failed/Gray**       | **Player unable to reserve the final sliver of health/no health left in the health bar**. The player can continue playing and if [Sudden Death (SD)][Sudden Death wikilink]/[Perfect (PF)][Perfect wikilink] was not used, refilling the health bar completely will "revive" the player from the failed status to normal. Also, score will not be contributed towards the team score at this state. Text colour will be changed from white to red.
+**Tag/Green**         | _Tag Coop and Tag Team Vs only_. **This is a permanent colour scheme for the match to all and will not change based on health bar status**. There will be an arrow (`play-warningarrow`) pointing the current player playing the beatmap and given a small white glow.
```

Comment:
also, this has been unconfirmed by Haskorion, but is the `arrow-generic` the other image that does this too?